src/core_ocean/shared/mpas_ocn_init_routines.F

```diff
@@ -93,7 +93,7 @@ module ocn_init_routines
 !>  This routine initializes max level and boundary mask variables
 !
 !-----------------------------------------------------------------------
-subroutine ocn_init_routines_compute_max_level(domain)!{{{
+subroutine ocn_init_routines_compute_min_max_level(domain)!{{{
```

Comment:
The corresponding `end subroutine ocn_init_routines_compute_min_max_level` change is needed.